connector/src/main/java/org.apache.flink/connector/nebula/sink/NebulaBatchOutputFormat.java

```diff
@@ -42,14 +41,14 @@
     protected final NebulaGraphConnectionProvider graphProvider;
     protected final OptionsT executionOptions;
     protected NebulaBatchExecutor<T> nebulaBatchExecutor;
-    private volatile AtomicLong numPendingRow;
+    private transient long numPendingRow;
```

Comment:
I changed the data type because I think there is no need to use an atomic value here, since all the methods using this value is `synchronized`.

Reply:
Yeah, it's right.